<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Twitter Moderator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTableView" name="TwitterFeed"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="AutoValidationLayout">
      <item>
       <widget class="QCheckBox" name="AutoValidation">
        <property name="text">
         <string>Automatic Validation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="AutoValidationTempo">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>300</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="AutoValidationTempoLabel">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="AutoValidationReset">
        <property name="text">
         <string>Reset Tempo</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="TwitterSearch">
        <property name="text">
         <string notr="true">#bae</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="TwitterSearchButton">
        <property name="text">
         <string>Apply Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1079</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSet_TwitterWall_Ip"/>
    <addaction name="separator"/>
    <addaction name="actionClear_Tweets"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Content_Panel"/>
    <addaction name="separator"/>
    <addaction name="actionDisconnect_User"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="ResetTweetView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Return to Tweet</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionActive_ReTweet">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Active ReTweet</string>
   </property>
  </action>
  <action name="actionShow_Content_Panel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Content Panel</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionDisconnect_User">
   <property name="text">
    <string>Disconnect User</string>
   </property>
  </action>
  <action name="actionSet_TwitterWall_Ip">
   <property name="text">
    <string>Set TwitterWall Ip</string>
   </property>
  </action>
  <action name="actionClear_Tweets">
   <property name="text">
    <string>Clear Tweets</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>AutoValidationTempo</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>AutoValidationTempoLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>556</x>
     <y>514</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
